src/gei/Handlers/GenerateScriptCommandHandler.cs

```diff
@@ -338,6 +340,8 @@ private string GenerateSequentialAdoScript(IDictionary<string, IEnumerable<strin
         content.AppendLine(VersionComment);
         content.AppendLine(EXEC_FUNCTION_BLOCK);
 
+        content.AppendLine("$RepoMigrations = [ordered]@{}");
```

Comment:
Recording this allows us to know what migrations generated an ID, and therefore what migrations we should wait for.